doc/recognition.xml

```diff
@@ -402,7 +402,7 @@ recognised by the current recognition node to the image or
 kernel of the found homomorphism:
 
 <ManSection>
-<Attr Name="forkernel" Arg="ri"/>
+<Attr Name="InitDataForKernelRecogNode" Arg="ri"/>
```

Comment:
I was confused by this name as to me `init` stands for the verb `initialize`, not for `initial` or `initialization`. And that makes for some confusing reading. So perhaps
```suggestion
<Attr Name="InitialDataForKernelRecogNode" Arg="ri"/>
```

Reply:
I changed the names accordingly.